### ISSUE TITLE
data-rellax-percentage doesn't work

### ISSUE THREAD

**Issue by soniapello** [dixonandmoe/rellax#49]
Hi I've try to center element with data-rellax-percentage but it doesn't work 
http://codepen.io/anon/pen/QpVpxE
Do I made mistake ?

**Comment by p-realinho** [comment]
Hey @soniapello , thanks for trying out Rellax :)

So, imagine you have a parent container, and you want to parallax it's content, but, you need the content to align with the parent container, when the parent container is scrolled into view.

The center/percentage options solve this issue.
When `center` is enabled, or the element has the `data-rellax-percentage` attribute, with a value of `0.5`, the content will align with it's parent container when it (the parent) reaches the center (`0.5`) of the viewport.
Playing with the value of the `data-rellax-percentage` will make them align at different points of the viewport.

So, in other words, it doesn't "center" the element, just dictates if and where it should align with it's parent.

Although there's no demo yet, there are a few implementations of this in the `tests` folder, that might illustrate better what I'm trying to explain. (`center.html` seems to be broken, but `percentage.html` for example is working fine)

Hope that clears things up a bit for you. If not, let me know, maybe I'll be able to help with a specific use case.

**Comment by soniapello** [comment]
Hi @p-realinho thanks for the explanation
That means that if I give a data-rellax-percentage of 0 the content will align with it's parent container at the bottom of the viewport ?
